spec.md

```diff
@@ -287,11 +287,11 @@ Valid values for `state` are `in progress`, `succeeded`, and `failed`. The platf
 
 The request provides these query string parameters as useful hints for brokers.
 
-|  Query-String Field | Type  | Description  |
+| Query-String Field | Type | Description |
 |---|---|---|
-| service_id  |  string | ID of the service from the catalog.  |
-| plan_id  | string  | ID of the plan from the catalog.  |
-| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client SHOULD provide the same value using this query parameter as a URL-encoded string.  |
+| service_id | string | ID of the service from the catalog. |
+| plan_id | string | ID of the plan from the catalog. |
+| operation | string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client SHOULD provide the same value using this query parameter as a URL-encoded string. |
```

Comment:
SHOULD for operation (the broker client SHOULD) is being changed to MUST in another PR, I assume it's going to get changed here as a merge?